txnkv/transaction/prewrite.go

```diff
@@ -365,6 +365,7 @@ func (action actionPrewrite) handleSingleBatch(c *twoPhaseCommitter, bo *retry.B
 		}
 		start := time.Now()
 		msBeforeExpired, err := c.store.GetLockResolver().ResolveLocks(bo, c.startTS, locks)
+		defer c.store.GetLockResolver().ResolveLocksDone(c.startTS)
```

Comment:
Why is the `defer` written in prewrite instead of in `ResolveLocks`? `ResolveLocksDone` looks missing if resolve lock is triggered by other commands.

Reply:
> Why is the `defer` written in prewrite instead of in `ResolveLocks`?

It's because we need to keep this infomation during backoffing.

> ResolveLocksDone looks missing if resolve lock is triggered by other commands.

I've added them now.